libraries/disp/viewers/hpisettingsview.cpp

```diff
@@ -242,6 +253,7 @@ void HpiSettingsView::loadSettings()
     m_pUi->m_checkBox_useComp->setChecked(settings.value(m_sSettingsPath + QString("/HpiSettingsView/useCOMP"), false).toBool());
     m_pUi->m_checkBox_continousHPI->setChecked(settings.value(m_sSettingsPath + QString("/HpiSettingsView/continousHPI"), false).toBool());
     m_pUi->m_doubleSpinBox_maxHPIContinousDist->setValue(settings.value(m_sSettingsPath + QString("/HpiSettingsView/maxError"), 10.0).toDouble());
+    m_pUi->m_spinBox_samplesToFit->setValue(settings.value(m_sSettingsPath + QString("/HpiSettingsView/fittingWindowSize"), 1000).toInt());
```

Comment:
It looks like here the default is set to 1000, whereas in the constexpr at the top of the file you set the default to 100. Is this intentional?

Reply:
Nope. I'll push a fix. It is a by product of me + the fact that some settings for a model are updated/initialized in the view!?